modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/integration.dm

```diff
@@ -8,8 +8,11 @@
 	var/datum/antagonist/bloodsucker/bloodsuckerdatum = IS_BLOODSUCKER(exposed_mob)
 	if(!bloodsuckerdatum)
 		return ..()
-	bloodsuckerdatum.bloodsucker_blood_volume = min(bloodsuckerdatum.bloodsucker_blood_volume + round(reac_volume, 0.1), BLOOD_VOLUME_MAXIMUM)
-
+	if(istype(bloodsuckerdatum.my_clan, /datum/bloodsucker_clan/ventrue) && bloodsuckerdatum.bloodsucker_blood_volume >= BLOOD_VOLUME_NORMAL)
```

Comment:
BLOODSUCKER_BLOOD_RANKUP_COST for ventrue is 550 blood, BLOOD_VOLUME_NORMAL is 560, they can still abuse blood drinking to get levels